src/local_scheduler/local_scheduler.cc

```diff
@@ -28,6 +28,9 @@
 #include "state/object_table.h"
 #include "state/error_table.h"
 
+// Using flatbuffer types defined in "format/local_scheduler_generated.h"
+using namespace ray::local_scheduler::protocol;
```

Comment:
Can we not use "using" and instead qualify uses? You can introduce an alias like `local_scheduler_protocol` as below.

Reply:
I'd just spell out `ray::local_scheduler::protocol` everywhere. If we insist on shortening it, then it should be `using local_scheduler_protocol  = ray::local_scheduler::protocol;`.